lighthouse-core/audits/byte-efficiency/uses-request-compression.js

```diff
@@ -40,8 +41,12 @@ class ResponsesAreCompressed extends ByteEfficiencyAudit {
   static audit_(artifacts) {
     const uncompressedResponses = artifacts.ResponseCompression;
 
+    function skipUnrelatedResponses(record) {
+      return URL.getProtocol(record.url) !== CHROME_EXTENSION_PROTOCOL;
+    }
+
     const results = [];
-    uncompressedResponses.forEach(record => {
+    uncompressedResponses.filter(skipUnrelatedResponses).forEach(record => {
```

Comment:
Now that I'm reading the audit I realize this actually just processes what's already been compressed over in the gatherer. 

Let's filter it [there](https://github.com/GoogleChrome/lighthouse/blob/master/lighthouse-core/gather/gatherers/dobetterweb/response-compression.js#L28) instead so we don't waste cycles re-compressing something we don't care about. Sorry for the bad tip in the issue!